### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by Kcchouette**
## Changes

* Modify the `i18n/fr.json` file

### CODE REVIEW

File: i18n/fr.json [hunk 1]
```diff
@@ -93,7 +93,7 @@
     "Anonymous":
         "Anonyme",
     "Avatar generated from IP address":
-        "Avatar anonyme (Vizhash de l'adresse IP)",
+        "Avatar généré à partir de l'adresse IP)",
```
Comment (r4sas) [inline i18n/fr.json]: Last ")" left.

**Comment by rugk** [comment]
Thanks for the review, @r4sas. Could you do a final review/approve the changes, if everything is okay now?

**Comment by r4sas** [comment]
In the technical part, everything is correct. But about changes in language itself - I do not know French.

**Comment by Draky50110** [comment]
Sorry, not here often...
Don't know how to download the new version to check FR translation...

**Comment by rugk** [comment]
Ah okay, @r4sas, so you just spotted the `)`. Thanks anyway…

@Draky50110 Would it be sufficient to see the changes for now? You can do so by switching to the ["Files changed" tab](https://github.com/PrivateBin/PrivateBin/pull/250/files) at the top.

Of course you can always review all translations, but you can still do that afterwards. :smiley: If you want to do so, the [file is here](https://github.com/PrivateBin/PrivateBin/blob/master/i18n/fr.json), you can even edit it online.

**Comment by Draky50110** [comment]
Ok, seems ok for me for FR :)

**Pull request merged**